apps/prairielearn/src/migrations/20250822211535_enrollments__joined_at__add.test.sql

```diff
@@ -0,0 +1,16 @@
+-- BLOCK select_enrollment_for_user_in_course_instance
```

Comment:
Unfortunately, there isn't a good way to type migration testing, since all model/types assume that the codebase is in a state where all migrations have occurred. Instead, I need to run some untyped queries here.